definitions/output/reports/tech_crux.js

```diff
@@ -0,0 +1,297 @@
+const pastMonth = constants.fnPastMonth(constants.currentMonth)
+
+publish('tech_crux', {
+  schema: 'reports',
+  type: 'incremental',
+  protected: true,
+  bigquery: {
+    partitionBy: 'date',
+    clusterBy: ['geo', 'client', 'rank', 'technology'],
+    requirePartitionFilter: true
+  },
+  tags: ['tech_report'],
+  dependOnDependencyAssertions: true
+}).preOps(ctx => `
+DELETE FROM ${ctx.self()}
+WHERE date = '${pastMonth}';
+
+CREATE TEMP FUNCTION IS_GOOD(
+  good FLOAT64,
+  needs_improvement FLOAT64,
+  poor FLOAT64
+) RETURNS BOOL AS (
+  SAFE_DIVIDE(good, good + needs_improvement + poor) >= 0.75
+);
+
+CREATE TEMP FUNCTION IS_NON_ZERO(
+  good FLOAT64,
+  needs_improvement FLOAT64,
+  poor FLOAT64
+) RETURNS BOOL AS (
+  good + needs_improvement + poor > 0
+);
+`).query(ctx => `
+WITH pages AS (
+  SELECT
+    client,
+    page,
+    root_page AS origin,
```

Comment:
Think I said this before but root_page (with trailing slash) != origin (without trailing slash). So we should either keep the root_page name or convert it to an actual origin.